src/client/Client.cc

```diff
@@ -9785,6 +9787,14 @@ int Client::statfs(const char *path, struct statvfs *stbuf,
 
   client_lock.Unlock();
   int rval = cond.wait();
+
+  if (Client::root == NULL) {
```

Comment:
I wrote Client::root in my comment to be specific, but in the code you can just say `root` to refer to a member